SwiftyJSONAccelerator/Base Files/ObjectMapperTemplate.txt

```diff
@@ -0,0 +1,15 @@
+        /**
```

Comment:
<img width="525" alt="screen shot 2015-10-25 at 08 37 22" src="https://cloud.githubusercontent.com/assets/193717/10714533/b821c2e0-7af3-11e5-9446-ca190d102a9f.png">
The tab is there in the generation string (hence would make it okishly aligned). I agree they are not perfect.
Old version:
<img width="438" alt="screen shot 2015-10-25 at 08 38 58" src="https://cloud.githubusercontent.com/assets/193717/10714539/f54b2a58-7af3-11e5-821a-f36e1a0cd985.png">